testsuite/test_cim_obj.py

```diff
@@ -2411,7 +2411,7 @@ class Test_CIMInstanceName_from_instance(object):
             ),
             dict(
                 classname='CIM_Foo',
-                keybindings={('P1', 'Ham'), ('P2', 'Cheese')}
+                keybindings={'P1': 'Ham', 'P2': 'Cheese'}
```

Comment:
If maintaining the order of keybinding items is important, this line should be changed to:
```
keybindings=[('P1', 'Ham'), ('P2', 'Cheese')]
```

Reply:
Done